planet/Objects.ocd/Environment.ocd/Stalactite.ocd/Script.c

```diff
@@ -31,8 +31,8 @@ private func Hit()
 	{
 		Size = PV_KeyFrames(0, 0, 0, 100, PV_Random(3, 5), 1000, 3),
 		R = (colour >> 16) & 0xff,
-		G = (colour >>  8) & 0xff,
-		B = (colour >>  0) & 0xff,
+		G = (colour >>	8) & 0xff,
+		B = (colour >>	0) & 0xff,
```

Comment:
In the future, please refrain from mixing format fixes for lines you usually wouldn't touch into a commit. That can really mess up merges big time - I remember when someone randomly renamed a variable and added parameter types in an unrelated commit and messed up my Controls branch :[
If you want to change formatting, do a separate commit (and in this case using tabs for non-indention formatting actually conflicts with our style guidelines)